build.rs

```diff
@@ -456,7 +457,13 @@ print(json.dumps({
     "base_prefix": base_prefix,
     "shared": PYPY or bool(sysconfig.get_config_var('Py_ENABLE_SHARED')),
     "executable": sys.executable,
-    "machine": platform.machine()
+
+    # The reason we don't use platform.architecture() here is that it's not
```

Comment:
I don't want to pass this string to Python.
How about moving to Rust side or just removing it(since we have the same one below)?

Reply:
Removed.